data/src/test/java/org/apache/iceberg/data/TestLocalScan.java

```diff
@@ -103,132 +103,100 @@ public TestLocalScan(String format) {
+      genericRecord.copy(ImmutableMap.of("id", 6L, "data", "fetta"))
+  );
+  private final List<Record> file2SecondSnapshotRecords = Lists.newArrayList(
+      genericRecord.copy(ImmutableMap.of("id", 14L, "data", "radical")),
+      genericRecord.copy(ImmutableMap.of("id", 15L, "data", "collocation")),
+      genericRecord.copy(ImmutableMap.of("id", 16L, "data", "book"))
+  );
+  private final List<Record> file3SecondSnapshotRecords = Lists.newArrayList(
+      genericRecord.copy(ImmutableMap.of("id", 24L, "data", "cloud")),
+      genericRecord.copy(ImmutableMap.of("id", 25L, "data", "zen")),
+      genericRecord.copy(ImmutableMap.of("id", 26L, "data", "sky"))
+  );
+
+  private final List<Record> file1ThirdSnapshotRecords = Lists.newArrayList(
+      genericRecord.copy(ImmutableMap.of("id", 6L, "data", "brainy")),
+      genericRecord.copy(ImmutableMap.of("id", 7L, "data", "film")),
+      genericRecord.copy(ImmutableMap.of("id", 8L, "data", "fetta"))
+  );
+  private final List<Record> file2ThirdSnapshotRecords = Lists.newArrayList(
+      genericRecord.copy(ImmutableMap.of("id", 16L, "data", "cake")),
+      genericRecord.copy(ImmutableMap.of("id", 17L, "data", "intrinsic")),
+      genericRecord.copy(ImmutableMap.of("id", 18L, "data", "paper"))
+  );
+  private final List<Record> file3ThirdSnapshotRecords = Lists.newArrayList(
+      genericRecord.copy(ImmutableMap.of("id", 26L, "data", "belleview")),
+      genericRecord.copy(ImmutableMap.of("id", 27L, "data", "overview")),
+      genericRecord.copy(ImmutableMap.of("id", 28L, "data", "tender"))
+  );
 
   private void overwriteExistingData() throws IOException {
-    Record record = GenericRecord.create(SCHEMA);
-
-    this.file1FirstSnapshotRecords = Lists.newArrayList(
-        record.copy(ImmutableMap.of("id", 4L, "data", "obscure")),
-        record.copy(ImmutableMap.of("id", 5L, "data", "secure")),
-        record.copy(ImmutableMap.of("id", 6L, "data", "fetta"))
-    );
-    DataFile file11 = writeFile(sharedTableLocation, format.addExtension("file-11"), file1FirstSnapshotRecords);
-
-    this.file2FirstSnapshotRecords = Lists.newArrayList(
-        record.copy(ImmutableMap.of("id", 14L, "data", "radical")),
-        record.copy(ImmutableMap.of("id", 15L, "data", "collocation")),
-        record.copy(ImmutableMap.of("id", 16L, "data", "book"))
-    );
-    DataFile file21 = writeFile(sharedTableLocation, format.addExtension("file-21"), file2FirstSnapshotRecords);
-
-    this.file3FirstSnapshotRecords = Lists.newArrayList(
-        record.copy(ImmutableMap.of("id", 24L, "data", "cloud")),
-        record.copy(ImmutableMap.of("id", 25L, "data", "zen")),
-        record.copy(ImmutableMap.of("id", 26L, "data", "sky"))
-    );
-    DataFile file31 = writeFile(sharedTableLocation, format.addExtension("file-31"), file3FirstSnapshotRecords);
+    DataFile file12 = writeFile(sharedTableLocation, format.addExtension("file-11"), file1SecondSnapshotRecords);
```

Comment:
It's a bit confusing that the name of the `DataFile` variable is different from the extension name, e.g. in this case `DataFile` is `file12` and extension `file-11`. Is there a reason for the rename from how it was before?

Reply:
I think this renaming produces some changes down below too.

Reply:
Nice catch, fixed the extension. 

The first snapshot is actually consisted of `file1FirstRecords`, `file2FirstRecords` and `file3FirstRecords`, so I renamed this to match the actual snapshot order.